comments/classified.csv

```diff
@@ -2074,18 +2023,8 @@ Wouldn't this be a wrong type for the call when ``#code`` is nullable? It feels
 Can we do some unification so RID is resolved consistently from a single source of truth for ngen/perfmap steps as well? Maybe not part of this PR but if it is now feasible I think it would smooth things up for new platforms if resolving RID in multiple places is avoided,0,0.5
 "some modifiers cause us to bail out of local function parsing and try member parsing, but generally we reject bad local function modifiers here",0,0.5
 "(edit: I had the wrong line number before, sorry)",0,0.5
-"IEquatable<IntPtr> [](start = 27, length = 18)",0,0.5
 "Given that we have a custom code to match the type, I think we should cover negative cases as well, i.e",0,0.5
 "- The name match, but containing namespace name doesn't",0,0.5
-#NAME,0,0.5
-- IEquatable is a nested type and container's name is and is not System,0,0.5
-#NAME,0,0.5
-#NAME,0,0.5
-- IEquatable has arity 2,0,0.5
-"- IEquatable is constructed with a ""wrong"" argument",0,0.5
-#NAME,0,0.5
-#NAME,0,0.5
-- Etc. #Closed,0,0.5
```

Comment:
All the ones that say `#NAME` we should delete, but maybe the others are worth keeping?

`- Etc. #Closed` can probably remove.

Reply:
Fixed!